README.md

```diff
@@ -259,6 +259,37 @@ CustomApp.getInitialProps = async (ctx: AppContext) => {
 
 <br />
 
+# ⚗️ CLI (experimental)
+
+You can use `unleash [action] [options]` in you `package.json` `scripts` section, or with:
+
+```sh
+npx @unleash/nextjs
+```
+
+Use [Environment Variables](#environment-variables) (you needed: `UNLEASH_SERVER_API_URL`, `UNLEASH_SERVER_API_TOKEN`), or shorthand versions: `UNLEASH_URL`, `UNLEASH_TOKEN`, `UNLEASH_APPNAME`.
+
+## Usage
+
+- `get-definitions <outputFile.json>` Download feature flags definitions for bootstrapping (offline use) of server-side SDK.
+- `generate-types [options] <outputFile.ts>` Generate types and typed functions from feature flags defined in an Unleash instance.
+  It will also generate strictly typed versions of `useFlag`, `useVariant`, `useFlags` and `flagsClient` (unless `--typesOnly` is used).
+  - `-t, --typesOnly` don't include typed versions of functions exported from `@unleash/nextjs` (default: false)
+  - `-b, --bootstrap <sourceFile.json>` load definitions from a file instead of fetching definitions - work offline
+- `-V` Output the version number
+
+## Example
+
+Try it now
+
+```sh
+UNLEASH_URL=https://app.unleash-hosted.com/demo/api \
+UNLEASH_TOKEN=test-server:default.8a090f30679be7254af997864d66b86e44dcfc5291916adff72a0fb5 \
+npx @unleash/nextjs generate-types ./unleash.ts
+```
+
+<br/>
```

Comment:
I let a few of these slip through previously, but I personally don't like using manual breaks in the markup. Much like with HTML, I don't think the text should try and define how it's presented. So I suggest we remove this one. 

That said, I'm more than happy to let you make a case for why we should keep them if you think we should.

```suggestion
```

Reply:
@Tymek I noticed you resolved this one and removed all the breaks. But it seems you might have forgotten this one? 😅